tests/browser/test/procedure_test.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Node.js script to run Automated tests in Chrome, via webdriver.
+ */
+
+const webdriverio = require('webdriverio');
+const chai = require('chai');
+const path = require('path');
+const {posixPath} = require('../../../scripts/helpers');
+
+let browser;
+suite('Testing Connecting Blocks', function (done) {
+  // Setting timeout to unlimited as the webdriver takes a longer time to run than most mocha test
+  this.timeout(0);
+
+  // Setup Selenium for all of the tests
+  suiteSetup(async function () {
+    const options = {
+      capabilities: {
+        'browserName': 'chrome',
+        'goog:chromeOptions': {
+          args: ['--allow-file-access-from-files'],
+        },
+      },
+      services: [['selenium-standalone']],
+      logLevel: 'warn',
+    };
+
+    // Run in headless mode on Github Actions.
+    if (process.env.CI) {
+      options.capabilities['goog:chromeOptions'].args.push(
+        '--headless',
+        '--no-sandbox',
+        '--disable-dev-shm-usage'
+      );
+    } else {
+      // --disable-gpu is needed to prevent Chrome from hanging on Linux with
+      // NVIDIA drivers older than v295.20. See
+      // https://github.com/google/blockly/issues/5345 for details.
+      options.capabilities['goog:chromeOptions'].args.push('--disable-gpu');
+    }
+    // Use Selenium to bring up the page
+    const url =
+      'https://rachel-fenichel.github.io/blockly/demos/code/index.html';
```

Comment:
Can this be tested on the playground rather than the code demo? Perferably using the same local testing as the [playground test](https://github.com/google/blockly/blob/d3939f7a488c4bd7868237a7899a67e98bc0a79d/tests/browser/test/basic_playground_test.js#L47).

Reply:
There isn't a way, that I know of, to run the procedures. I think this is why all procedure testing uses 'https://rachel-fenichel.github.io/blockly/demos/code/index.html' since it has the run button built in

Reply:
@cpcallen Can we use the local code demo here instead of Rachel's hosted code demo?